backend/entities/Pims.Dal.Entities.csproj

```diff
@@ -11,6 +11,10 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer.NetTopologySuite" Version="5.0.6" />
     <PackageReference Include="NetTopologySuite" Version="2.2.0" />
     <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="5.0.6">
```

Comment:
I think only the entity project was using the design part of EF. So it got moved to that project